src/main/ts/api.ts

```diff
@@ -73,3 +118,22 @@ export function getIssues() {
     return chartData;
   });
 }
+
+export function getProjects({ q }: { q: string }) {
```

Comment:
`q` can be empty, even maybe undefined. Would be best to avoid passing it during the call in that case, to prevent such calls from happening
`http://localhost:9000/api/projects/search?q=`